tf/edgeml/trainer/bonsaiTrainer.py

```diff
@@ -26,6 +25,7 @@ def __init__(self, bonsaiObj, lW, lT, lV, lZ, sW, sT, sV, sZ,
         '''
 
         self.bonsaiObj = bonsaiObj
+        self.regression_Loss = regLoss
```

Comment:
@siddharthdivi , please use self.regressionLoss. It needs to be camel case, not _. Please fix this.

@metastableB , this has to be resolved before merge.